share/man/man7/c.7

```diff
@@ -349,15 +453,17 @@ C Standard.
 .%A ISO/IEC
 .%T 9899:2018 (aka C17)
 .Re
+.Pp
+.Rs
+.%A ISO/IEC
+.%T 9899:2024 (aka C23)
+.Re
 .Sh HISTORY
 This manual page first appeared in
 .Fx 9.0 .
 .Sh AUTHORS
 .An -nosplit
 This manual page was originally written by
-.An Gabor Kovesdan Aq Mt gabor@FreeBSD.org .
-It was updated for
-.Fx 14.0
-by
-.An Faraz Vahedi Aq Mt kfv@kfv.io
-with information about more recent C standards.
+.An Gabor Kovesdan Aq Mt gabor@FreeBSD.org
+and is now maintained by
+.An Faraz Vahedi Aq Mt kfv@kfv.io .
```

Comment:
We don't generally claim maintainership in the documentation.

Reply:
Makes sense, I'll revert it then, alright.